### PR TITLE
Hook into event-tickets to inject the event date into the ticket email

### PR DESCRIPTION
**Pull request by borkweb**
Related: https://github.com/moderntribe/event-tickets/pull/83

See: https://central.tri.be/issues/41992

### CODE REVIEW

File: src/Tribe/Event_Tickets/Main.php [hunk 1]
```diff
@@ -56,4 +66,19 @@ public function attendees_report( $object = null ) {
 
 		return $this->attendees_report;
 	}
+
+	/**
+	 * Ticket email integration class object accessor method
+	 *
+	 * @since 4.0.2
+	 */
```
Comment (barryhughes) [inline src/Tribe/Event_Tickets/Main.php]: `@return`?

File: src/Tribe/Event_Tickets/Main.php [hunk 2]
```diff
@@ -24,6 +24,13 @@ class Tribe__Events__Event_Tickets__Main {
 	private $attendees_report;
 
 	/**
+	 * Contains an instance of the Ticket Email integration class
+	 * @since 4.0.2
```
Comment (barryhughes) [inline src/Tribe/Event_Tickets/Main.php]: I do enjoy a bit of space between the description and the @tags but I realize that's slightly subjective :-D

**Comment by barryhughes** [comment]
Looks good to me, will leave it to your discretion if you want to address any of the above before merging :-)

**Comment by borkweb** [comment]
All good comments. Fixed and added a few `@param` tags.

**Comment by barryhughes** [comment]
:cake:

**Pull request merged**